src/Resources/config/services.yml

```diff
@@ -9,7 +9,7 @@ services:
         arguments:
             - "@service_container"
         tags:
-            - { name: kernel.event_listener, event: kernel.controller, method: onKernelController, priority: 254 }
+            - { name: kernel.event_listener, event: kernel.controller, method: onKernelRequest, priority: 31 }
```

Comment:
I have changed all priorities accordingly in 36ba667a261dee51f830e0237af523aab1419220